lib/rubysl/rubysl-socket/lib/socket/constants.rb

```diff
@@ -24,9 +24,10 @@ module Constants
 
     # MRI defines these constants manually, thus our FFI generators don't pick
     # them up.
-    EAI_ADDRFAMILY = 1
-    EAI_NODATA = 7
-    IPPORT_USERRESERVED = 5000
+    # Truffle: added unless const_defined? to avoid warnings
+    EAI_ADDRFAMILY = 1 unless const_defined?(:EAI_ADDRFAMILY)
```

Comment:
Would this syntax be nicer? `EAI_ADDRFAMILY ||= 1`

Reply:
Seems like it would be harder to notice if we update this code.
This should get fixed properly with another PR cleaning up the rbx constants anyway.